docs/extensions/listings/shortcodes-and-filters.yml

```
@@ -169,6 +169,13 @@
     Output nicely formatted versions of fancy strings such as LaTeX and BibTeX
     in multiple formats.
 
+- name: figure-table-formatter
+  path: https://github.com/iagopinal/figure-table-formatter
+  author: [Iago Pinal-Fernandez](https://github.com/iagopinal)
```

```suggestion
  author: '[Iago Pinal-Fernandez](https://github.com/iagopinal)'
```

Sorry I missed the quotes